internal/runners/reset/reset.go

```diff
@@ -0,0 +1,96 @@
+package reset
+
+import (
+	"github.com/ActiveState/cli/internal/locale"
+	"github.com/ActiveState/cli/internal/output"
+	"github.com/ActiveState/cli/internal/primer"
+	"github.com/ActiveState/cli/internal/prompt"
+	"github.com/ActiveState/cli/internal/runbits"
+	"github.com/ActiveState/cli/pkg/cmdlets/commit"
+	"github.com/ActiveState/cli/pkg/platform/authentication"
+	"github.com/ActiveState/cli/pkg/platform/model"
+	"github.com/ActiveState/cli/pkg/project"
+	"github.com/go-openapi/strfmt"
+)
+
+type Reset struct {
+	out     output.Outputer
+	auth    *authentication.Auth
+	prompt  prompt.Prompter
+	project *project.Project
+	config  configurable
+}
+
+type configurable interface {
+	CachePath() string
+}
+
+type primeable interface {
+	primer.Outputer
+	primer.Auther
+	primer.Prompter
+	primer.Projecter
+	primer.Configurer
+}
+
+func New(prime primeable) *Reset {
+	return &Reset{
+		prime.Output(),
+		prime.Auth(),
+		prime.Prompt(),
+		prime.Project(),
+		prime.Config(),
+	}
+}
+
+func (r *Reset) Run() error {
+	if r.project == nil {
+		return locale.NewInputError("err_no_project")
+	}
+
+	latestCommit, err := model.BranchCommitID(r.project.Owner(), r.project.Name(), r.project.BranchName())
+	if err != nil {
+		return locale.WrapError(err, "err_reset_latest_commit", "Could not get latest commit ID")
+	}
+	if *latestCommit == r.project.CommitUUID() {
+		return locale.NewInputError("err_reset_latest", "You are already on the latest commit")
+	}
+
+	r.out.Print(locale.Tl("reset_commit", "Your project will be reset at [ACTIONABLE]{{.V0}}[/RESET]\n", latestCommit.String()))
```

Comment:
```suggestion
	r.out.Print(locale.Tl("reset_commit", "Your project will be reset to [ACTIONABLE]{{.V0}}[/RESET]\n", latestCommit.String()))
```

Let's include the date and author of this commit, since the ID alone isn't very helpful.